<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAccountForm</class>
 <widget class="QWidget" name="AddAccountForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="../../../../res/resources.qrc">
        <normaloff>:/icons/hand_16px.ico</normaloff>:/icons/hand_16px.ico</iconset>
      </attribute>
      <attribute name="title">
       <string>Manuellement</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButtonAddAccount">
         <property name="text">
          <string>Ajouter</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../res/resources.qrc">
           <normaloff>:/icons/plus_16px.ico</normaloff>:/icons/plus_16px.ico</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBoxOptions">
         <property name="title">
          <string>Options avancées</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>De :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>À :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBox_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Vous devez ajouter &lt;span style=&quot; font-weight:600;&quot;&gt;%&lt;/span&gt; dans le nom de compte. &lt;br&gt;Par exemple, pour générer &lt;span style=&quot; font-style:italic;&quot;&gt;nom_de_compte&lt;/span&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;1&lt;/span&gt; à &lt;span style=&quot; font-style:italic;&quot;&gt;nom_de_compte&lt;/span&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;5&lt;/span&gt;, vous devez écrire&lt;span style=&quot; font-style:italic;&quot;&gt; nom_de_compte&lt;/span&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;%&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Identifiants de connexion</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditAccount"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Mot de passe :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditPassword">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Nom de compte :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="../../../../res/resources.qrc">
        <normaloff>:/icons/laptop_16px.ico</normaloff>:/icons/laptop_16px.ico</iconset>
      </attribute>
      <attribute name="title">
       <string>Dynamiquement</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="5" column="3">
        <widget class="QPushButton" name="pushButtonRefreshNickname">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../res/resources.qrc">
           <normaloff>:/icons/arrow_refresh.ico</normaloff>:/icons/arrow_refresh.ico</iconset>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Pseudo :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1" rowspan="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Nom de compte :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="lineEditEmail"/>
       </item>
       <item row="8" column="1" colspan="3">
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Email :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="lineEditAccountName"/>
       </item>
       <item row="8" column="4">
        <widget class="QPushButton" name="pushButtonRefreshGlobal">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../res/resources.qrc">
           <normaloff>:/icons/arrow_refresh.ico</normaloff>:/icons/arrow_refresh.ico</iconset>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QPushButton" name="pushButtonLockNickname">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../res/resources.qrc">
           <normaloff>:/icons/lock.ico</normaloff>:/icons/lock.ico</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="4">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Console</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QTextEdit" name="textEditConsole">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="pushButtonRefreshPassword">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../res/resources.qrc">
           <normaloff>:/icons/arrow_refresh.ico</normaloff>:/icons/arrow_refresh.ico</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="3" rowspan="2">
        <widget class="QPushButton" name="pushButtonRefreshAccountName">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../res/resources.qrc">
           <normaloff>:/icons/arrow_refresh.ico</normaloff>:/icons/arrow_refresh.ico</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QPushButton" name="pushButtonLockPassword">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../res/resources.qrc">
           <normaloff>:/icons/lock.ico</normaloff>:/icons/lock.ico</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QPushButton" name="pushButtonLockEmail">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../res/resources.qrc">
           <normaloff>:/icons/lock.ico</normaloff>:/icons/lock.ico</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="4" rowspan="2">
        <widget class="QPushButton" name="pushButtonLockAccountName">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../res/resources.qrc">
           <normaloff>:/icons/lock.ico</normaloff>:/icons/lock.ico</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="pushButtonRefreshEmail">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../res/resources.qrc">
           <normaloff>:/icons/arrow_refresh.ico</normaloff>:/icons/arrow_refresh.ico</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="lineEditPassword_2"/>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="lineEditNickname"/>
       </item>
       <item row="6" column="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Mot de passe :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="4">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButtonSettings">
           <property name="text">
            <string>Options</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../res/resources.qrc">
             <normaloff>:/icons/cog_16px.ico</normaloff>:/icons/cog_16px.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonValidate">
           <property name="text">
            <string>Valider</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../res/resources.qrc">
             <normaloff>:/icons/tick_16px.ico</normaloff>:/icons/tick_16px.ico</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEditAccount</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>groupBoxOptions</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>spinBox_2</tabstop>
  <tabstop>pushButtonAddAccount</tabstop>
  <tabstop>lineEditAccountName</tabstop>
  <tabstop>lineEditPassword_2</tabstop>
  <tabstop>lineEditEmail</tabstop>
  <tabstop>lineEditNickname</tabstop>
  <tabstop>pushButtonRefreshAccountName</tabstop>
  <tabstop>pushButtonRefreshPassword</tabstop>
  <tabstop>pushButtonRefreshEmail</tabstop>
  <tabstop>pushButtonRefreshNickname</tabstop>
  <tabstop>pushButtonLockAccountName</tabstop>
  <tabstop>pushButtonLockPassword</tabstop>
  <tabstop>pushButtonLockEmail</tabstop>
  <tabstop>pushButtonLockNickname</tabstop>
  <tabstop>textEditConsole</tabstop>
  <tabstop>pushButtonRefreshGlobal</tabstop>
  <tabstop>pushButtonSettings</tabstop>
  <tabstop>pushButtonValidate</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
